NewWorkbench/NewWorkbench/Content/plugins/fullcalendar/fullcalendar.css

```diff
@@ -0,0 +1,589 @@
+/*!
+ * FullCalendar v1.6.4 Stylesheet
+ * Docs & License: http://arshaw.com/fullcalendar/
+ * (c) 2013 Adam Shaw
+ */
+
+
+.fc {
+	direction: ltr;
+	text-align: left;
+	}
+
+.fc table {
+	border-collapse: collapse;
+	border-spacing: 0;
+	}
+
+html .fc,
+.fc table {
+	font-size: 1em;
+	}
+
+.fc td,
+.fc th {
+	padding: 0;
+	vertical-align: top;
+	}
+
+
+
+/* Header
+------------------------------------------------------------------------*/
+
+.fc-header td {
+	white-space: nowrap;
+	}
+
+.fc-header-left {
+	width: 25%;
+	text-align: left;
+	}
+
+.fc-header-center {
+	text-align: center;
+	}
+
+.fc-header-right {
+	width: 25%;
+	text-align: right;
+	}
+
+.fc-header-title {
+	display: inline-block;
+	vertical-align: top;
+	}
+
+.fc-header-title h2 {
+	margin-top: 0;
+	white-space: nowrap;
+	}
+
+.fc .fc-header-space {
+	padding-left: 10px;
+	}
+
+.fc-header .fc-button {
+	margin-bottom: 1em;
+	vertical-align: top;
+	}
+
+/* buttons edges butting together */
+
+.fc-header .fc-button {
+	margin-right: -1px;
+	}
+
+.fc-header .fc-corner-right,  /* non-theme */
+.fc-header .ui-corner-right { /* theme */
+	margin-right: 0; /* back to normal */
+	}
+
+/* button layering (for border precedence) */
+
+.fc-header .fc-state-hover,
+.fc-header .ui-state-hover {
+	z-index: 2;
+	}
+
+.fc-header .fc-state-down {
+	z-index: 3;
+	}
+
+.fc-header .fc-state-active,
+.fc-header .ui-state-active {
+	z-index: 4;
+	}
+
+
+
+/* Content
+------------------------------------------------------------------------*/
+
+.fc-content {
+	clear: both;
+	zoom: 1; /* for IE7, gives accurate coordinates for [un]freezeContentHeight */
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this usage of the experimental "zoom" property. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=css%3Aexperimental-property-usage)